ci/azure/windows.yml

```diff
@@ -20,20 +20,23 @@ jobs:
     - powershell: |
         Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
         Write-Host "##vso[task.prependpath]$HOME/miniconda3/bin"
+        Write-Host "##vso[task.prependpath]$HOME/miniconda3/condabin"
       displayName: 'Add conda to PATH'
-    - script: conda update -q -n base conda
-      displayName: Update conda
-    - script: |
-        call activate
+    - bash: |
+        conda update -q -y -n base conda
+      displayName: 'Update conda'
+    - bash: |
+        source $HOME/miniconda3/etc/profile.d/conda.sh
+        conda activate pandas-dev
         conda env create -q --file ci\\deps\\azure-windows-$(CONDA_PY).yaml
       displayName: 'Create anaconda environment'
     - script: |
-        call activate pandas-dev
-        call conda list
+        call conda.bat activate pandas-dev
```

Comment:
Why not just change all of the `script` commands to `bash` and then just use without `call`? Right now the usage looks mixed so would make things consistent and easier to read I think for non-Windows people

Reply:
changed all of `windows.yml` file `script` steps to use `bash` instead